src/server/api/routes/api-router.js

```diff
@@ -31,4 +32,6 @@ router.use('/documentation', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 // Application routes
 router.use('/modules', modulesRouter);
 
+router.use('/channel_members', channelMembersController);
```

Comment:
Why is this route `channel_members` when the title of the task is `channel-members`?